src/main/java/codersafterdark/reskillable/base/LevelLockHandler.java

```diff
@@ -42,44 +43,27 @@
     private static Map<Class<?>, List<Class<? extends LockKey>>> lockTypesMap = new HashMap<>();
     private static Map<LockKey, Set<FuzzyLockKey>> fuzzyLockInfo = new HashMap<>();
     private static String[] configLocks;
```

Comment:
Can be removed, same with the DEFAULT_SKILL_LOCKS a few lines up but it is not letting me mark it.